csrc/scheduler/matmul.cpp

```diff
@@ -646,16 +713,44 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
     }
   }
 
-  // [Mo, No, Ko, Mi, Ni, Ki]
-  // Propagate tiling globally
-  scheduler_utils::transformPropagateToAllFrom(cc, -1);
+  if (params.split_k_factor > 1) {
```

Comment:
A minor suggestion:

Perhaps we can have a const flag defined at the beginning of `scheduleMatmul`:
```
const bool has_split_k = params.split_k_factor > 1;
```
and use it in the conditions. This will improve a little bit code readability but the key value is that the check will be in a single place (and later it may be modified).